plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/page/SeoPageDAO.java

```diff
@@ -36,48 +36,50 @@
 
 /**
  * Data Access Object for the 'page' objects
+ *
  * @author E.Santoboni
  */
 public class SeoPageDAO extends PageDAO {
 
-	private static final Logger _logger =  LoggerFactory.getLogger(SeoPageDAO.class);
+    private static final Logger _logger = LoggerFactory.getLogger(SeoPageDAO.class);
+
+    @Override
+    protected PageExtraConfigDOM getExtraConfigDOM() {
```

Comment:
A "get" method that creates a new object is a bad pattern. We should rename this to `createExtraConfigDom()`